data/report_templates/project_summary.md

```diff
@@ -160,9 +184,8 @@ UPPNEX project, which was created for you when your order was placed:
 {{ project.UPPMAX_path }}
 ```
 
-
-If you have problems accessing your data, please contact SciLifeLab
-[genomics_support@scilifelab.se](mailto: genomics_support@scilifelab.se).
+If you have problems accessing your data, please contact NGI
+[{{ project.support_email }}](mailto: {{ project.support_email }}).
```

Comment:
yes, it works :D but will change if that gives you peace (as a fellow `OCD`er :stuck_out_tongue_winking_eye: )